docs/git-compatibility.md

```diff
@@ -109,6 +93,62 @@ https://github.com/octocat/Hello-World` will clone GitHub's "Hello-World" repo
 into a directory by the same name.
 
 
+## Co-located Jujutsu/Git repos
+
+A "co-located" Jujutsu repo is a hybrid Jujutsu/Git repo. These can be created
+if you initialize the Jujutsu repo in an existing Git repo by running `jj init
+--git-repo=.` or with `jj git clone --colocate`. The Git repo and the Jujutsu
+repo then share the same working copy. Jujutsu will import and export from and
+to the Git repo on every `jj` command automatically.
+
+This mode is very convenient useful when tools (e.g. build tools) expect a Git
+repo to be present.
+
+It is allowed to mix `jj` and `git` commands in such a repo in any order.
+However, it may be easier to keep track of what is going on if you mostly use
+read-only `git` commands and use `jj` to make changes to the repo. If you need
+to use several mutating `git` commands in a row, you may have to first `git
+switch` to a branch, and it may be easier to avoid using `jj` commands until the
+task you needed `git` for is finished.
```

Comment:
Good point, but maybe we should explain why it's that way? I mean that we should explain that any `jj` commands will result in "detached HEAD" mode. Maybe something like this (but feel free to rephrase):

> It is allowed to mix `jj` and `git` commands in such a repo in any order. Note that many `jj` commands leave the Git repo in a "detached HEAD" state. For this reason, it may be easier...

Reply:
Good point, I tried to rewrite it. Actually, I was mostly thinking about problems like the ones I described below in the "downsides" section, but there's probably no actual need to repeat that.